docs/grading_lists/single_list_grader.md

```diff
@@ -9,7 +9,7 @@ grader = SingleListGrader(
 )
 ```
 
-To receive full points for this problem, a student would enter `cat, dog` or `dog, cat` into the input box. Entering `cat, octopus` or just `cat` will receive half points.
+To receive full points for this problem, a student would enter `cat, dog` or `dog, cat` into the input box. Entering `cat, octopus` or just `cat` will receive half points. The `answers` key is a python list of the individual answers you wish to require, and is a required key for `SingleListGrader`.
```

Comment:
Is this true? I would have expected that 

```xml
<script type="loncapa/python">
from mitxgraders import SingleListGrader
grader = SingleListGrader()
</script>
<customresponse cfn='grader' expect="['cat', 'dog']">
    
</customresponse>
```
since `SingleListGrader` is an `ItemGrader`.

Reply:
The expect key is passed as a string, whereas `SingleListGrader` requires a list. It doesn't know how to cast that string to a list.